### ISSUE TITLE
How to nuke all Fraidycat data and start over?

### ISSUE THREAD

**Issue by solarchemist** [kickscondor/fraidycat#234]
**Before you start.**
* [X] I have checked the [Common Problems](https://github.com/kickscondor/fraidycat/wiki/Common-Problems) page.
* [X] I have searched the issue tracker for a bug that matches the one I want to file, without success.

**Desktop (please complete the following information):**
 - OS: Ubuntu Linux 18.04.5
 - Browser: Firefox 78.15.0esr
- Fraidycat 1.1.10 (Firefox extension, while logged in to Firefox Sync)

Fraidycat clearly has an impressive system for preserving its feeds, because even after 

+ removing the Firefox extension, or
+ removing the Firefox extension and deleting all files with the string `fraidy` anywhere in their filename inside the Firefox profile directory, or
+ removing the Firefox extension, logging out from Firefox Sync, visually inspecting `extensions.json` and getting the extension ID string, then deleting all local folders/files referencing said ID: `find . -name "*94060031*" -exec rm -r {} +`, or
+ deleting all [local storage from the "Inspect" view](https://github.com/kickscondor/fraidycat/wiki/Spelunking-Fraidycat's-Save-Files) in Firefox's `about:debugging` window, then removing the extension,

the feeds and all tags all still there when I reinstall the Firefox extension.

I added a bunch of feeds using the OPML import feature (which worked great by the way), but unfortunately they did not end up in the tag I had open, and instead polluted the "Home" tab. There is simply too many of them to change their tag one-by-one, so I exported JSON and edited the JSON file (after deminifying it, by changing the `tags` field for all feeds using some simple search-and-replace matches) then reminified the JSON and imported that into Fraidycat.
But it seems this import had no effect (I was hoping it would overwrite all my existing feeds and tags).

So, does anyone have an idea on how I can reset Fraidycat to a blank slate, so I can start over?

**Comment by solarchemist** [comment]
I just realised this issue is a duplicate of #78. Sorry about that.

Note that although #78 is closed, the problem of wiping and starting over with Fraidycat remains unresolved. Any suggestions would be very welcome!

**Comment by fasiha** [comment]
I removed the extension, signed out of Firefox Sync, and then [refreshed Firefox](https://support.mozilla.org/en-US/kb/refresh-firefox-reset-add-ons-and-settings). When I installed Fraidycat, it doesn't remember my old stuff. Hooray!

**Comment by solarchemist** [comment]
I wasn't aware of the refresh feature in Firefox. Very neat.

Unfortunately (for me) I have long since manually re-organised my Fraidycat feeds, and so I haven't tested your suggestion myself. But this does indeed sound like it resolves this issue, and I hope many future Fraidycat users can benefit from more easily starting over using this method. Thank you @fasiha! 

Closing this issue as fixed.

**Comment by insanity54** [comment]
I found a different way of deleting Fraidycat settings without having to do a Firefox refresh.

First go to the extension debugging screen by typing  `about:debugging#/runtime/this-firefox` in the URL bar.

Find Fraidycat in the extensions list and click the Inspect button.

In the developer tools that pops up, click the Storage tab.

In the panel on the left side, there's a list of different types of storages. Cache Storage, Cookies, Extension Storage, etc. 

Expand Extension Storage and there will be a moz-extension://<uuid> in the list. Right click on that and click delete all. Then, remove the extension and re-install.

**Comment by solarchemist** [comment]
Very nice! Thanks for reporting here despite the issue being closed!

Can confirm this resurrected a Fraidycat instance on Firefox that was perpetually stuck on the "LOADING" screen for weeks (I had resigned myself to using Chromium instead). And it worked without even having to restart Firefox :-) And all the feeds and categories were still there after following the steps in @insanity54 post.

I think this issue can be definitely closed now ;-)

The following issues in this repo reported similar problems and might benefit from @insanity54's method:

+ https://github.com/kickscondor/fraidycat/issues/262
+ https://github.com/kickscondor/fraidycat/issues/169
+ https://github.com/kickscondor/fraidycat/issues/230

**Comment by EdoSecco** [comment]
> First go to the extension debugging screen by typing `about:debugging#/runtime/this-firefox` in the URL bar.
> 
> Find Fraidycat in the extensions list and click the Inspect button.
> 
> In the developer tools that pops up, click the Storage tab.
> 
> In the panel on the left side, there's a list of different types of storages. Cache Storage, Cookies, Extension Storage, etc.

Since my Fraidycat on Firefox suddenly stopped working (and it isn't the first time!!) I tried to go there and i found... nothing.

It seems all the archive has been wiped out with no good reason. As said, it happened before.
Worst of all, when I reinstalled the addon, it didn't upload anything, as if **it wasn't Firefox-synced**!!!

So frustrating, especially since there isn't an automatic setting for periodical export of the json/opml.

This really makes me want to find another similar addon and abandon Fraidycat, and I'm very sorry about this since I really liked it for years.
